lib/_tls_common.js

```diff
@@ -116,6 +117,20 @@ exports.createSecureContext = function createSecureContext(options, context) {
     }
   }
 
+  if (options.pskCallback && c.context.enablePskCallback) {
+    c.context.onpskexchange = (identity) => {
+      var ret = options.pskCallback(identity);
+      if (ret) {
+        assert(typeof ret === 'object');
```

Comment:
Not sure how important it is, but this will be a fatal exception if it will fail (i.e. node will exit).